general/administration/hardware-acceleration.md

```diff
@@ -49,6 +49,117 @@ The hardware acceleration is available immediately for media playback. No server
+
+6. IMPORTANT!!! YOU NEED TO RUN THESE COMMANDS IN CLI NOT DE NOT DE TERMINAL CLI 
+        To do this we need to go out of GUI(Desktop Enviroment) and into CLI(kernal recovery terminal). 
+        So we do this by Pressing CTRL+ALT+F1 (Some people DE(desktop enviroment) will already be on F1 so change it to any other F#
+
+7. Now your should see a terminal login with your user
+          IE. my user is ```devilscoder```
+          so i loing with user ```devilscoder``` and the password associated with it.
+
+9. Now make the this directory and navigate to it by sending this command.
+        ```sudo mkdir /opt/nvidia && cd /opt/nvidia```
+
+10. Now we need to download the correctly update driver from the command posted below
+        ```sudo wget https://international.download.nvidia.com/XFree86/Linux-x86_64/(driver firmware number)/NVIDIA-Linux-x86_(driver firmware number).run```
+        IE if my drivers firmware revision is 335.21 that we found on the nvidia driver search on hte nvidia website. So mine would look like this
+        ```sudo wget https://international.download.nvidia.com/XFree86/Linux-x86_64/435.21/NVIDIA-Linux-x86_64-435.21.run```
+
+11. Now we need to give them permissions.
+        ```sudo chmod +x ./NVIDIA-Linux-x86_64-(driver firmware number).run```
+
+12.Now we need to run the driver installer.
+           ```./NVIDIA-Linux-x86_64-430.50.run```
+
+14. Make sure the new drivers is installed and recognized by typing in
+            ```nvidia-smi```
+
+15. Once the driver is installed we now need to install and patch the driver with the patch file downloaded from the github.
+            ```cd ~/```
+            ```sudo git clone https://ipfs.io/ipns/Qmed4r8yrBP162WK1ybd1DJWhLUi4t6mGuBoB9fLtjxR7u nvidia-patch```
+            ```bash ./patch.sh```
+16. Now it should display succesfully patched. Now that everythings done we can restart xServer or whatever you were using for DE (Desktop Enviroment)    
+            ```startx```
+            Your Desktop Enviroment should then display login and move your way to your jellyfin install in browser
+            ie. (internal ip adress of server):8096 ie. 192.168.1.155:8096
+            ie. (if your on the computer where jellyfin is installed)       http://localhost:8096 
+            ie. (if you have a reverse-proxy running) https://sub.yourdomain.com        https://stream.jellyfin.com
+17. Once there click on the top left hamburger menu and click on dashboard in the menu
+
+18. Once Loaded click on the hamburger menu again and select playback
+
+19. Now is IMPORTANT!!! Open this link [NVENC/DECODING SETTINGS FOR GPU](https://developer.nvidia.com/video-encode-decode-gpu-support-matrix#Encoder)
+                Once there if your card is 2 years or older it will most likely not have NVENC (NVIDIA ENCODING) on the card. So lets go  scroll to the bottom and select the green button that says Geoforce/Titan becuase mine in this example is a Geoforce GTX 970.
```

Comment:
I'd leave out the numbers my 5y old can do NVENC just fine

Reply:
I know but it builds hte backbone strcuture of the composition. Meaning if i remove the numbers the entire formatting scheme will be messed up.

Reply:
WDYM? Just remove the "2 years" part, the rest is fine. Also 2y wouldn't be accurate in 1 year anyway and no one will probably update this number weed year